deliverables/README.md

```diff
@@ -8,61 +8,154 @@ To answer questions, please just put the answer on the next line after the quest
 
 Example:  
 
-**What is your name?**
-Sir Student of Ruby
+**What is your name?**   
+Jamie Finney
 
-**What is your quest?**
-I seek the Rails  
+**What is your quest?**    
+Maek the money don't let the money make me.  
 
-**What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+**What is your favorite color?**  
+Green!  
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+**What is the role of HTML in a web page?**   
+HTML is the skeleton of the page.
+
+**What is the role of CSS in a web page?**    
+CSS provides the styling for the page.
+
+**What is the role of Javascript in a web page?**   
+Javascript powers the interface of a page.
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?**    
+HTML is static text/content on a page, while CSS styles and organizes the HTML and around it.  CSS is a much more arganized and faster way to style HTML rather than in the HTML file.
```

Comment:
Correct.  The reason it's more organized is the separation of concerns.  You put all the CSS in one file, it makes it easier to get an overall idea of what's going on with styling.